frontend/src/App.js

```diff
@@ -3,18 +3,21 @@ import TopBar from "./components/TopBar";
 import MainPage from "./components/MainPage";
 import Footer from "./components/Footer";
 import { Switch, Route } from "react-router-dom";
-import MoviePageFilterByTitle from "./components/MoviePageFilterByTitle";
+import MovieSheet from "./components/MovieSheet";
 import "./App.css";
-import ListMovies from "./components/ListMovies";
+import ListMoviesByGenre from "./components/ListMoviesByGenre";
 
 function App() {
   return (
     <div className="App">
       <TopBar />
       <Switch>
         <Route exact path="/" component={MainPage} />
-        <Route path="/movieSheet" component={MoviePageFilterByTitle} />
-        <Route path="/listMovies/:genreName" component={ListMovies} />
+        <Route path="/movieSheet" component={MovieSheet} />
+        <Route
+          path="/listMoviesByGenre/:genreName"
```

Comment:
`/genre/:genreName` would be a shorter/better path for this route, don't you think ?

Reply:
.... don't really know what to say. We think the code is easier to understand if path has same same as component isn't?